biz.ganttproject.core/build.gradle

```diff
@@ -29,6 +29,15 @@ artifacts {
     cloud jar
 }
 dependencies {
+    implementation 'junit:junit:4.12'
+    implementation 'junit:junit:4.12'
+    implementation 'junit:junit:4.12'
+    implementation 'junit:junit:4.12'
+    implementation 'junit:junit:4.12'
+    implementation 'junit:junit:4.12'
+    implementation 'org.junit.jupiter:junit-jupiter:5.4.2'
+    implementation 'org.junit.jupiter:junit-jupiter:5.4.2'
+    implementation 'org.junit.jupiter:junit-jupiter:5.4.2'
```

Comment:
That's a little bit too much, huh? Also, it shall not be in "implementation"

Reply:
Don't even remember adding those Oo